sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -308,4 +313,48 @@ export class StudentController extends BaseController {
 
     stream.pipe(response);
   }
+
+  @Get("application-summary")
+  async getStudentApplicationSummary(
+    @UserToken() userToken: IUserToken,
+  ): Promise<StudentApplicationDTO[]> {
+    const existingStudent = await this.studentService.getStudentByUserName(
```

Comment:
Now that we have the student id on token and we have a method in the student service to getStudentByUserId, I think that we should start to use the search by id (primary key) instead of the username, IMO.